### ISSUE TITLE
Error: download.geofabrik.de on 127.0.0.11:53: no such host

### ISSUE THREAD

**Issue by iray-tno** [openmaptiles/openmaptiles#801]
Hello!

While I'm running `quickstart.sh`, I got an error on `download-geofabrik`.

```
-------------------------------------------------------------------------------------
====> : Downloading testdata japan.osm.pbf
mkdir -p build && mkdir -p data && mkdir -p cache
=============== download-geofabrik =======================
Download area: japan
[[ example: make download-geofabrik area=albania ]]
[[ list areas: make download-geofabrik-list ]]
docker-compose run --rm -u $(id -u ${USER}):$(id -g ${USER}) import-osm ./download-geofabrik.sh japan
Creating network "openmaptiles_postgres_conn" with driver "bridge"
Creating volume "openmaptiles_pgdata" with default driver
DEBUG[0006] request URL: https://download.geofabrik.de/ failed with response: &{0 [] 0xc0000f79b0 0xc000142400 <nil>} error=Get https://download.geofabrik.de/: dial tcp: lookup download.geofabrik.de on 127.0.0.11:53: no such host
INFO[0006] /import/geofabrik.yml generated.
FATAL[0000] can't find japan error=japan is not in config. Please use "list" command
make: *** [Makefile:95: download-geofabrik] エラー 1
```

Full log -> [quickstart.log](https://gist.github.com/iray-tno/a4141f5d3233c793b084b3c58c0c069a)

I'm using Docker for windows via cygwin on WIndows 10.
And I'm in company proxy. Configured proxy stuff in settings of Docker Desktop for Windows.

(I hesitated to open a new issue because my situation is very specific and there are some issues about similar problems, but my error looks different.)

Thanks.

**Comment by smellman** [comment]
@iray-tno It seems dns issue.

```
DEBUG[0006] request URL: https://download.geofabrik.de/ failed with response: &{0 [] 0xc0000f79b0 0xc000142400 <nil>} error=Get https://download.geofabrik.de/: dial tcp: lookup download.geofabrik.de on 127.0.0.11:53: no such host
```

```
dial tcp: lookup download.geofabrik.de on 127.0.0.11:53: no such host
```

Please setup your docker environment to support dns lookup.

**Comment by iray-tno** [comment]
@smellman Thank you for quick reply.

I'm setting DNS Server to 8.8.8.8 in Docker Desktop for windows. (Is this the correct way?)
But still getting the same error.

**Comment by iray-tno** [comment]
I manually downloaded the .pbf file and put it in the data folder.
And then I got an error same with #784 . I had to comment out `import-wikidata`.
Its looks like it is working fine. (still in processing)

I thought it was not network problem since 'docker run hello-world' works fine. 🤔

**Comment by smellman** [comment]
@iray-tno Manually download will fail in generate tiles because download-geofabrik task generate `data/docker-compose-config.yml`, it is used generate tiles process.

Also, `docker run hello-world` command isn't correct to research your network in docker container.
Please run `docker run -it --rm busybox` and run `ping download.geofabrik.de` inside docker container.

**Comment by iray-tno** [comment]
@smellman Thanks again. That was what I wanted to know.

I tried as you suggested. (I also tried wget since the proxy only allows http(s))

```
> ping download.geofabrik.de
ping: bad address 'download.geofabrik.de'

> wget download.geofabrik.de
wget: 'download.geofabrik.de'
```

It seems that was my configuration problem. And I couldn't fix it..
Well.., I guess I can bring `docker-compose-config.yml` there from out of the proxy. 🤓

This issue can be closed.

**Comment by nyurik** [comment]
the geofabrik downloader has been removed from tools, and published as 4.1 release.  Please use `download-osm` utility instead (part of tools).  I will close this issue, but please comment and/or reopen if there any issues.

**Comment by iray-tno** [comment]
Thanks.

By the way, I've solved this problem by setting proxy for container (not for docker command) in `%USERPROFILE%\.docker\config.json`:

```json
{
  "credsStore": "desktop",
  "stackOrchestrator": "swarm",
  "proxies": {
    "default": {
      "httpProxy": "http://[address]:[port]",
      "httpsProxy": "http://[address]:[port]",
      "noProxy": "localhost"
    }
  }
}
```

It was not a DNS issue, but a proxy issue.
And It's confusing that the configuration of proxy in Docker Desktop UI doesn't affect connection from a container inside.
In Windows, You have to edit `%USERPROFILE%\.docker\config.json` file to set up the proxy.

Additionally, I recommend using the IP address of the DNS server, not the host name.
